Libraries/gnustep/libs-gui_NSSound.patch

```diff
@@ -0,0 +1,11 @@
+diff -ruN a/Headers/AppKit/NSSound.h b/Headers/AppKit/NSSound.h
+--- a/Headers/AppKit/NSSound.h	2022-12-29 04:05:31.000000000 -0500
++++ b/Headers/AppKit/NSSound.h	2023-10-02 22:43:36.772505741 -0500
+@@ -31,6 +31,7 @@
+ #ifndef _GNUstep_H_NSSound
+ #define _GNUstep_H_NSSound
+ 
++#import <AppKit/AppKitDefines.h>
```

Comment:
Is it necessary? AFAIK libs-gui compiles without this change.

Reply:
The same compilation error occurs and to fix it apply the same corrective that was approved.

https://github.com/gnustep/libs-gui/commit/f10c0eeb1ba5276d32c70a479f77eff54c6bdba5

![Screenshot 2023-10-04 at 6 58 59 AM](https://github.com/trunkmaster/nextspace/assets/3104815/12184ccf-78d4-490b-ac84-866a014e5481)

Reply:
Hm... It's strange. I see no problem here. Let's remove this patch from this PR and see how CircleCI build will go. If it will fail for the same reason - I check it again.

Reply:
You're right it's broken libs-gui release. I'll add a separate commit with appropriate description.